include/vkgcDefs.h

```diff
@@ -267,8 +277,7 @@ struct PipelineOptions {
 
   ShadowDescriptorTableUsage shadowDescriptorTableUsage; ///< Controls shadow descriptor table.
   unsigned shadowDescriptorTablePtrHigh;                 ///< Sets high part of VA ptr for shadow descriptor table.
-  bool nullDescriptor;                                   ///< If set, support VK_EXT_robustness2 to give defined
-                                                         ///  behavior for null descriptor
+  ExtendedRobustness extendedRobustness;                 ///< If set, support VK_EXT_robustness2 features.
```

Comment:
what's the difference between PipelineOptions::robustBufferAccess and PipelineOptions::ExtendedRobustness::robustBufferAccess? could you use a different name if they have difference?

Reply:
PipelineOptions::ExtendedRobustness::robustBufferAccess represents the feature robustBufferAccess2 in VK_EXT_robustness2, which has more sticker requirements of out-of-bounds check than robustBufferAccess core feature.

Reply:
Please adjust the comments slightly to make this clearer.

PipelineOptions::robustBufferAccess should say:

> If set, out of bounds accesses to buffer or private array do not cause a fault, but may read or write data anywhere in the buffer or array.

PipelineOptions::ExtendedRobustness::robustBufferAccess should say:

> Whether buffer accesses are tightly bounds-checked against the range of the descriptor. Give defined behavior (e.g., read 0) for out-of-bounds buffer access and descriptor range rounding up.

By making these adjustments, we give the reader a high-level idea of the distinction. It is okay to refer them to the Vulkan spec for more detail. For example, a comment could say that ExtendedRobustness is intended to correspond to the features of VK_EXT_robustness2.

Reply:
Thanks. Fixed.